package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@aragon/core",
+  "name": "@aragon/osx",
```

Comment:
This will collide with the package.json under `packages/contracts`. I would remove it or use a different thing

Reply:
@mathewmeconry didn't merge https://github.com/aragon/osx/pull/265 yet and can make these changes himself.

Reply:
go ahead. I probably close #265 and start over